### ISSUE TITLE
Issue /is top command

### ISSUE THREAD

**Issue by banicola** [BentoBoxWorld/Level#153]
#### Describe the bug
When users try to use the /is top command there is an error message (generic error) in the chat and an error in the console (see the pastbin below)

#### Steps to reproduce the behavior
Use the command /is top with players in the top online (Last update)

#### Expected behavior
See the top 10 islands on the server

#### Screenshots and videos (Optional)
https://pastebin.com/x1f1y2PM

### Environment
Paper 1.15.2 Build BentoBoxWorld/BentoBox#284 (latest)

##### BentoBox Version (Mandatory)
Version 1.13.0

**Comment by tastybento** [comment]
Do `bbox version` in the console and share the output.

I will assume you are using the JSON default database. Share the contents of the files in the /plugins/BentoBox/database/TopTenData folder, e.g. bskyblock.json.

The issue is being caused by a request to get a head for a player with no name, which is weird...

**Comment by banicola** [comment]
![image](https://user-images.githubusercontent.com/25954399/82143022-0efd5100-9841-11ea-9ed6-20143a23ddc6.png)

We use MariaDB for our database.
`{
  "uniqueId": "bskyblock_world",
  "topTen": {
    "b50bf028-bbd3-43f9-acc4-2a65b490c733": 755,
    "a573a4a8-f4c2-491e-a1c1-ed1eccbb7aa4": 755,
    "a06ac440-9a2f-4d9b-91a7-42cc46d8ba41": 283,
    "cf4546a7-d593-4054-a9ea-7c46a70bd0a5": 33,
    "ab6ecf55-5200-40e8-a72e-3d3a559d5b91": 109,
    "9eb38ae0-d71e-48d5-9579-2f1b975e77f5": 4466,
    "38265b5f-7826-4ec9-a305-3f54ac24205e": 7483,
    "d22a4286-3c53-4c2b-9223-9a785bca2342": 2,
    "b21d1fee-03fc-4922-8e93-30b1738590ed": 14,
    "a044aff7-8468-465d-9337-3b43d3e816a1": 410,
    "3669647e-ca7c-4f0e-91a3-326b21e59c42": 3,
    "54a4a85e-fc45-411d-8008-03a2266a85ab": 3,
    "ed2f8459-4ec6-4b51-8ca2-59767d45af4c": 15173,
    "d9a58417-c0a6-44ef-950d-931de3fc32b8": 4807,
    "a4a4272a-d130-41d3-9e5a-e12ef0b2d977": 1629,
    "369983cd-27f6-4b94-ae2e-f0d0d6635c40": 3005,
    "77fc376d-4148-497e-be2d-fc3d6c957b35": 106,
    "ff222cce-922f-4a18-9ddd-36a2ae5e0c65": 5,
    "7ac7b737-8138-4efc-90fc-728e22a52673": 825,
    "a58e4bb1-8030-4a5b-9c8b-eb5f3c819bad": 687,
    "2d0abaca-1cce-4387-bb5e-ffecd773c5d8": 124,
    "928dba10-6b31-49f0-8b75-88e9de0fc333": 18009,
    "016afc7f-8b64-4345-90e5-7f78cf4cc940": 759,
    "90e20ebf-5b46-4dbd-b56b-44d9f4cfa217": 344,
    "5ae02f1c-3e1b-42fc-8da0-9822fd365346": 126,
    "769ec9b2-6ac5-4ec7-afcd-4a97bc65e394": 6,
    "58a15d9e-9121-481e-9952-5cfaa4995119": 1,
    "bf9f3bfa-0de5-4861-b3d9-4663ef179438": 5,
    "354f8f1d-03ab-4a62-9bf4-ead26c10ec82": 10625,
    "83840324-7aeb-4364-ae30-749de41d0fb6": 4,
    "3424c56f-58c5-4428-96de-cdb6c2997fff": 2643,
    "f2205a15-3bf2-45c3-bc5e-f8968b725d93": 749,
    "f1b06d8f-6a0e-4223-9e30-630396c04001": 2748,
    "7be5785c-36e9-4cfb-a9e5-c3c09f380188": 3,
    "a1945d68-03f6-413c-bd01-e426b825231f": 42,
    "1a758641-bfc7-446d-8e7b-aac14d821d78": 5038,
    "9ad39710-4666-45b7-8d02-7f1d2ccd9ec5": 374,
    "ecdb8d9a-77f0-4852-a919-2f0cadc9ed9d": 110,
    "ea30a62d-05fc-4c55-9205-b7fcde363f51": 120,
    "5e67f180-423c-4610-8587-39ef8fa2e780": 7,
    "7510e3c0-35cb-4e75-97b1-0bf55274bfd7": 86,
    "cac89981-f270-44a5-93c0-866112645c8f": 3759,
    "252b7e40-2de7-4dd2-ab58-821624de4b48": 3,
    "f7e6ed06-5c90-422b-9906-7505bb8c7b19": 16,
    "7dd0d187-6679-407b-bbba-f6231514c3b3": 62,
    "5db5aeaa-91ff-4b62-b5cd-bc55ce02defb": 8,
    "11c91674-3fe1-4a10-960f-566d7f815cc6": 9,
    "76107eb7-82b0-4ae6-9fcc-124aa99b3e88": 1,
    "c5d7b672-50b8-4a8d-b251-ca000db334b3": 21,
    "77ffacc4-2712-4f3b-b207-3c735e625c82": 2,
    "5d375785-c108-4f75-b3d4-153ac31f7b1c": 7,
    "ad59d2db-cfa0-4cac-875e-7346c987348e": 3,
    "040b20ef-dcf8-440c-b6b9-6443f795ba32": 3,
    "f2535472-6ab0-4b79-8a73-3bb6b4264a3c": 210,
    "5520d4ee-1833-431b-9852-c3c509a0f24e": 327,
    "718e9ab2-9984-405d-8781-30f05d4cea50": 21,
    "7883779c-2fe8-4dbb-b702-38164d4bba80": 417,
    "721c05ed-37ee-47e3-afdf-ed2ed852986d": 419,
    "757de464-edba-4765-804c-3674066cf7da": 119,
    "344a3a0d-a093-4811-908e-addeaaf99d69": 88,
    "d62e4ec7-09fa-4d74-9218-6944a7f0fcf2": 2,
    "a987adb0-b194-4d91-a04a-a823308fbbb7": 9,
    "d8b0fa33-92c5-405f-a3b7-047f85dace83": 7,
    "d9107019-0d04-4877-b4ec-71dce355ac2c": 2,
    "37978408-eac7-4670-bb82-804fa911d2ef": -1,
    "c031df42-4c68-483a-b78c-f658ae6efe7e": 10
  }
}`

**Comment by banicola** [comment]
I tried to set the permission bskyblock.island.top to false for all the players and I can do /is top as an admin.

But when I remove the permission or set it to true, the error comes back.

**Comment by tastybento** [comment]
This bug is coming from the server so it seems that it might be related to online/offline mode. I found some old bugs here:
https://bugs.mojang.com/browse/MC-50221

It says to start the server in online mode, then switch to offline mode.

I also found references that say the `usercache.json` file may be corrupted. Try to stop the server, save a backup of that file and restart.

The reason why the permission thing would have an affect is that it removes players and their UUIDs from the top ten, so their head is not requested or tried to be applied. So, I think this is a user cache issue with the server right now.
